### PR TITLE
Add OTF validator stream querying

### PR DESCRIPTION
**Pull request by dbobrenko**
## Changes
  - Add additional flag to parse validator's stream, as a part of Organic Scoring implementation

### CODE REVIEW

File: validators/sn1_validator_wrapper.py [hunk 1]
```diff
@@ -36,7 +36,10 @@ async def get_stream_response(self, params: QueryValidatorParams) -> StreamRespo
         # task_name = utils.guess_task_name(params.messages[-1])
 
         # Get the list of uids to query for this step.
-        uids = self.sample_uids(params)
+        if self._query_validators:
+            uids = sample_hotkey_uids(self.validator, [self.validator.wallet.hotkey.ss58_address], k=1)
```
Comment (Hollyqui) [inline validators/sn1_validator_wrapper.py:40]: is k=1 right to be fixed here?

**Review by Hollyqui — APPROVED** [review]
LGTM, check comment

File: validators/sn1_validator_wrapper.py (hunk 1 above)
Comment (steffencruz) [reply validators/sn1_validator_wrapper.py:40]: Maybe we can reuse the `get_random_uids` function to reduce repetition

**Review by steffencruz — APPROVED** [review]
LGTM

**Pull request merged**